sdk/cosmosdb/cosmos/.azure-pipelines/cosmos.test.yml

```diff
@@ -0,0 +1,55 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+trigger:
+  branches:
+    include:
+    - master
+    - cosmos-build-master
+  paths:
+    include:
+    - sdk/cosmosdb/cosmos/*
```

Comment:
I think this path should be `sdk/cosmosdb/cosmos` (without the trailing `/*`).  I believe this syntax should match everything recursively under the cosmos folder.  If you add the trailing `/*`, then it would only match files directly in the folder, not recursively in a subfolder.

Reply:
Thanks! I've removed the `*` and it looks working.